docs/tools/vdb_table/data/vespa.json

```diff
@@ -73,23 +73,23 @@
     "comment": ""
   },
   "embeddings_image": {
-    "support": "none",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/vespa-engine/sample-apps/tree/master/text-image-search",
     "comment": ""
   },
   "embeddings_structured": {
-    "support": "",
+    "support": "full",
     "source_url": "",
-    "comment": ""
+    "comment": "Through the support for custom ONNX models, Vespa also supports structured embeddings"
```

Comment:
I see. I guess in most cases, for structured embeddings, it would be desirable for users to define a custom model.
So, to understand better, a "full" support for this would mean an "out-of-the-box"-model that would e.g. combine text (embedding fields), categorical fields (e.g. product category) and numeric values (price, rating etc.) into one common embedding? 

Would the training need to be done online, or would it be OK with offline batched model updates?

Reply:
Yep something like that - if the user defines the fields and the DB infra trains, deploys and runs inference on that model upon ingestion and querying - that would count as "full".

Batch training vs online is whole another level of debate, let's say for now batch is enough :)

Does this sound reasonable?

Reply:
Absolutely!